includes/config.h

```diff
@@ -36,19 +36,19 @@
 #define PHW_FAULHABER 2
 
 /// Defines which preset to use for controlling pressure
-#define PNEUMATIC_HARDWARE_VERSION PHW_CHU
+#define PNEUMATIC_HARDWARE_VERSION PHW_FAULHABER
```

Comment:
So the `PHW_FAULHABER` is now the default pneumatic hardware version?

Reply:
Yes it is